doc/source/archiver.rst

```diff
@@ -0,0 +1,61 @@
+.. currentmodule:: databroker
+
+
+**********************************
+Connection with Archiver Appliance
+**********************************
+
+The Archiver Appliance is an EPICS logging system developed in a collaboration of SLAC, BNL, and MSU for collecting and storing measurements from various control devices. Data retrieval is based on the client-server interface using HTTP requests. Large-scale accelerators and experimental facilities may maintain multiple archivers. Within the databroker application, each archiver is represented by an ArchiverEventSource that needs to be configured and registred with the Broker. The following sections highlight two major topics: ArchiverEventSource configuration and data retrival based on the databroker interface.
+
+Configuration
+-------------
+
+ArchiverEventSources is configured via :class:`Broker` by extending the databroker
+configuration file with the archivers entry. For example, the following
+cxs_with_archiver.yml file illustrates the configuration of
+two ArchiverEventSources, arch_csx and arch_acc:
+
+.. code-block:: yaml
+
+    archivers:
+      module: databroker.eventsource.archiver
+      class: _ArchiverReader
```

Comment:
The config shouldn't need to refer to private classes. Whatever class is referenced here should be public.